source/operating-a-service/incident-playbook.html.md

```diff
@@ -64,43 +73,47 @@ Contact your users if:
 
 Informing users about incidents is generally considered best practice, but should be decided on a case by case basis with the product and service managers.
 
-### 8. If the incident requires invocation of Disaster Recovery procedures
+The Teacher Services team maintains a publicly available [service status dashboard](https://teacher-services-status.education.gov.uk/). For some incidents, you might find it helpful to update this. You will need a GitHub account to do this, or delegate updates to a colleague who has one.
 
-Follow the [Disaster Recovery procedure](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/disaster-recovery.md)
+The updates are managed via GitHub Actions and issues on the [teacher-services-upptime repository on GitHub](https://github.com/DFE-Digital/teacher-services-upptime). If a service’s automatic health check is failing continuously, an issue will be created within 5 minutes of the failure occurring and the dashboard will start reporting a service issue.
 
-## While the incident is in progress (all incident leads)
+To update the dashboard:
 
-Keep all conversations and status updates about the incident on the dedicated Slack incident channel.
+1. Navigate to the appropriate incident issue on the [GitHub issues page](https://github.com/DFE-Digital/teacher-services-upptime/issues)
+2. Add a comment to the issue
```

Comment:
This is quite a big change. Do we want this? In my experience, consolidating discussion in the Slack incident channel works pretty well and is delightfully accessible.